### PR TITLE
Python 2/3

### PR DESCRIPTION
**Pull request by jakirkham**
Combine the Python 2 and Python 3 builds back together again. Re-render with `conda-smithy` 2.4.0 to get matrix builds for CircleCI.

### CODE REVIEW

**Comment by jakirkham** [comment]
The diff here looks really weird for some reason. Locally I can only see the CI configuration file changes and the change to `meta.yaml`'s `skip`.

**Comment by jakirkham** [comment]
Merging with `master` seems to have fixed it.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ build:
     - vc9  # [win and py27]
     - vc10  # [win and py34]
     - vc14  # [win and py35]
```
Comment (jakirkham) [inline recipe/meta.yaml]: I don't think we need these as `python` is a dependency.

Comment (Korijn) [reply recipe/meta.yaml]: Go ahead and strip 'em?

Comment (jakirkham) [reply recipe/meta.yaml]: I'm ok with that. Thought we should save it for a subsequent PR though.

**Comment by jakirkham** [comment]
Ok, this should be good to go.

**Comment by jakirkham** [comment]
@Korijn, would it be alright if we merge this?

**Comment by Korijn** [comment]
Yes, for sure!

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Handling in PR ( https://github.com/conda-forge/vtk-feedstock/pull/41 ).

**Comment by jakirkham** [comment]
Should add that I deleted the `py2` branch as its contents are now included in `master` thanks to the merge commit here. So the history remains intact.

**Pull request merged**